tls/_common/prf.py

```diff
@@ -0,0 +1,69 @@
+# This file is dual licensed under the terms of the Apache License, Version
```

Comment:
What do you think about adding a docstring to this module that mentions Section 5 of RFC 5246, and maybe links to it too?